code/modules/mob/living/basic/farm_animals/cow/cow_wisdom.dm

```diff
@@ -4,6 +4,19 @@
 	desc = "Known for its wisdom, shares it with all."
 	gold_core_spawnable = FALSE
 	ai_controller = /datum/ai_controller/basic_controller/cow/wisdom
+	///The type of wisdom this cow will grant
+	var/granted_wisdom
+	///How much experience this cow will grant.
+	var/granted_experience
+
+/mob/living/basic/cow/wisdom/Initialize(mapload, granted_wisdom, granted_experience = 500)
```

Comment:
Software engineering is wild. Common assumptions is "Well, magic numbers are bad, should be named constants," but what do we meaningfully gain from `#define WISDOM_COW_DEFAULT_EXPERIENCE` in some defines.dm somewhere?

I _suppose_ if this 500 XP is based on something like "the XP required to gain Y levels of a skill" it should be in the same place as other skill defines, if they exist.

Reply:
500xp seems to be arbitrary from what I can tell? From 0 skill its enough to reach JOURNEYMAN but the gap between each skill level is larger than the last.